sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorDispatcher.java

```diff
@@ -50,6 +50,7 @@
     private final ConcurrentLinkedQueue<Work> workQueue;
```

Comment:
Please update to the changelog with this fix. :D

Reply:
Thanks, Connie, actually I'll move pr description to a GitIssue so that we can refer link from Changelog.